### DISCUSSION TITLE
Why bootstrap is using blue color as the primary color?

### DISCUSSION THREAD

**Discussion by rohit2sharma95** [twbs/bootstrap#30937] [General] [answered]
Is there any specific reason to make the blue color (the current primary color in bootstrap) as primary

Maybe if you use any other light and a little bit of shiny color. Like Github has a green color, it is pretty nice and cool. :nerd_face:

**Comment by mdo** [comment, marked as answer]
Blue is the default link color on the web, symbolizes stability/trust/etc, and doesn't come with too much implicit semantic value (e.g., green means success, red means danger, etc). It's a neutral-ish color to build on. Hope that helps!

**Reply by ffoodd** [reply]
A little historical background on ["Why are links blue?"](https://alistapart.com/blog/post/why-are-links-blue/) —basically, the darkest color after black in 16 colors displays— to also mention that blue is coincidently the color most perceptible by humans.

> Red and green are the colours most affected by colour-vision deficiency. Almost no one has a blue deficiency. Accordingly, nearly everyone can see blue, or, more accurately, almost everyone can distinguish blue as a colour different from others. It was pure good luck that the default colour of hyperlinks is blue with underlining.

Blue is good :)
